src/app/rundown-view/services/action-trigger-producer-key-binding.service.ts

```diff
@@ -35,7 +35,7 @@ export class ActionTriggerProducerKeyBindingService implements KeyBindingService
   private rundown?: Rundown
   private systemKeyBindings: KeyBinding[] = []
   private actionTriggerKeyBindings: KeyBinding[] = []
-  private readonly keyBindingsSubject: Subject<KeyBinding[]>
+  private keyBindingsSubject: Subject<KeyBinding[]>
```

Comment:
You are assigning this to be a behaviorSubject further down :D I propose this change instead:

```suggestion
  private keyBindingsSubject = new BehaviorSubject<KeyBinding[]>(this.systemKeyBindings)
```

And then you can delete line 59

Reply:
It is updated to suggestion :)